osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -77,6 +77,7 @@ protected override void InitialiseDefaults()
             Set(OsuSetting.BlurLevel, 0, 0, 1, 0.01);
 
             Set(OsuSetting.ShowInterface, true);
+            Set(OsuSetting.HideHealthBar, true);
```

Comment:
Please make this descriptive ie. `HideHealthBarWhenCantFail`

Reply:
Resolved.